.grabl/automation.yml

```diff
@@ -18,7 +18,7 @@
 build:
   correctness:
     build:
-      image: vaticle-ubuntu-21.04
+      image: vaticle-ubuntu-20.04
```

Comment:
Downgrade because this means the apt sources are still available for neo4j
To make the performance comparison fair, we have to use the same ubuntu version for TypeDB